bin/postinstall

```diff
@@ -10,33 +10,36 @@ CLI="${APP_NAME}"
 custom_gemfile=$(${CLI} config:get CUSTOM_PLUGIN_GEMFILE || echo "")
 if [ -n "$custom_gemfile" ] && [ -f "$custom_gemfile" ]; then
 
-	# Need to override the frozen setting for the vendored gems
-	${CLI} run bundle config --local frozen 0
+        # Need to override the frozen setting for the vendored gems
+        ${CLI} run bundle config --local frozen 0
 
-	# Re-bundle the application including gems
-	${CLI} run bundle install
+        # Re-bundle the application including gems
+        ${CLI} run bundle install
 
-	# Mark assets for recompilation in the OpenProject postinstall rake step
-	${CLI} config:set MUST_REBUILD_ASSETS="true"
+        # Mark backend for recompilation in the OpenProject postinstall rake step
+        echo "Custom Gemfile present. Need to recompile rails assets. Setting RECOMPILE_RAILS_ASSETS=true"
+        ${CLI} config:set RECOMPILE_RAILS_ASSETS="true"
 fi
 
-server_prefix=$(${CLI} config:get SERVER_PATH_PREFIX || echo "/")
-if [ -n "$server_prefix" ] && [ "$server_prefix" != '/' ]; then
-	${CLI} config:set MUST_REBUILD_ASSETS="true"
+# Check for server prefix
+server_prefix=$(${CLI} config:get SERVER_PREFIX || echo "")
+if [ "$server_prefix" != "/" ]; then
+        echo "Server prefix is set. Need to recompile rails assets. Setting RECOMPILE_RAILS_ASSETS=true"
+        # Mark backend for recompilation in the OpenProject postinstall rake step
+        ${CLI} config:set RECOMPILE_RAILS_ASSETS="true"
 fi
 
-
-# If we don't have to rebuild assets, set EXECJS_RUNTIME
-must_rebuild=$(${CLI} config:get MUST_REBUILD_ASSETS || echo "")
+# Check if we need to compile angular as well
+must_rebuild=$(${CLI} config:get RECOMPILE_ANGULAR_ASSETS= || echo "")
 if [ -n "$must_rebuild" ]; then
-	echo "Assets were requested to be rebuilt. Installing node dependencies..."
-	# Set execjs to node since that's installed
-	${CLI} config:set EXECJS_RUNTIME="Node"
-	openproject run bash -c "cd $APP_HOME && npm install"
-else
-        ${CLI} config:set EXECJS_RUNTIME="Disabled"
+        echo "RECOMPILE_ANGULAR_ASSETS was set. Installing node dependencies..."
+        openproject run bash -c "cd $APP_HOME && npm install"
```

Comment:
I think this could simply be writtent as:

```suggestion
        openproject run npm install
```

Reply:
I thought I ran into issues when I originally added this some time ago, but it looks to run fine.